java/upgrades/b4e689c90426b017ad550e30a439cab2763ff424/upgrade.properties

```
@@ -0,0 +1,2 @@
+description: Java 16: allow static interfaces
```

Shouldn't that be "local" since interfaces are always (implicitly) static:
```suggestion
description: Java 16: allow local interfaces
```